src/nsrt_learning/strips_learning/gen_to_spec_learner.py

```diff
@@ -457,16 +572,14 @@ def _get_pnads_with_keep_effects(
                     keep_effects_subset)
                 add_effects = pnad.op.add_effects | set(keep_effects_subset)
                 # Create the new PNAD.
-                new_pnad_op = pnad.op.copy_with(name=f"{pnad.op.name}-KEEP{i}",
-                                                parameters=parameters,
+                new_pnad_op = pnad.op.copy_with(parameters=parameters,
```

Comment:
so the idea is we're cool with them all being named the same now because we'll rename once at the very very end?